### PR TITLE
chore: Add mix check and mix assets.test tasks for convenience

### PR DESCRIPTION
**Pull request by joshlarson**
No ticket.

I just noticed I was running `mix test && mix credo && mix dialyzer --quiet && mix sobelow` a lot, and I wanted a single command to do it all.

### CODE REVIEW

**Comment by joshlarson** [comment]
I won't merge this before https://github.com/mbta/skate/pull/2328, because it's a branch off of a branch, but I'd love to get approval now so that I can just merge them one right after the other.

**Review by lemald — APPROVED** [review]

**Pull request merged**
